Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/InteractableRotationTheme.cs

```diff
@@ -12,7 +12,8 @@ namespace Microsoft.MixedReality.Toolkit.UI
     /// </summary>
     public class InteractableRotationTheme : InteractableThemeBase
     {
-        private Transform hostTransform;
+        protected Vector3 originalRotation;
+		protected Transform hostTransform;
```

Comment:
minor style comment to keep code consistent: please indent so that all members have same indentation level